README.md

```diff
@@ -160,7 +179,7 @@ To install extension-related Rake tasks, add this to your `Rakefile`:
 require 'solidus_dev_support/rake_tasks'
 SolidusDevSupport::RakeTasks.install
 
-task default: 'extension:specs'
+task default: %w[extension:test_app extension:specs]
```

Comment:
Is this expected? I thought we had removed `extension:test_app` from the default Rake task.